src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -90,6 +90,7 @@ public static class ScriptConstants
 
         public const string FeatureFlagDisableShadowCopy = "DisableShadowCopy";
         public const string FeatureFlagsEnableDynamicExtensionLoading = "EnableDynamicExtensionLoading";
+        public const string FeatureFlagEnableActionResult = "FeatureFlagEnableActionResult";
```

Comment:
The string value shouldn't have the `FeatureFlag` prefix. I suggest something like `EnableActionResultHandling`